release: v1.52.0 (#5719)
apollographql/router: .changesets/fix_bryn_service_version.md

> ### Allow overriding of service version ([PR #5689](https://github.com/apollographql/router/pull/5689))

Reviewer: ```suggestion
### Allow service version overrides ([PR #5689](https://github.com/apollographql/router/pull/5689))
```

> Previously `service.version` was not overridable via yaml and was ignored. It is now possible to set this explicitly which can be useful for users producing custom builds of the Router.

Reviewer: @BrynCooke - please review the edited changeset

```suggestion
The router now supports configuration of `service.version` via YAML file configuration. This enables users to produce custom versioned builds of the router.

```

> For example:

Reviewer: ```suggestion
The following example overrides the version to be `1.0`:
```

```yaml
telemetry:
  exporters:
    tracing:
      common:
        resource:
          service.version: 1.0
```

> Overrides the version to `1.0`.

Reviewer: ```suggestion

```
